oh_queue/static/js/common.js

```diff
@@ -43,6 +43,26 @@ function initializeTooltip(elem, options) {
   $(elem).tooltip(options);
 }
 
+function goto(state, url) {
+  if (state.currentUser.pranked) {
+      window.open(url, "_blank");
+  } else {
+        window.open("https://www.youtube.com/watch?v=dQw4w9WgXcQ");
+        app.makeRequest("pranked");
+        setTimeout(() => {
+            Swal.fire(
```

Comment:
Looks like a random notification if the browser is blocking popups

Reply:
Yeah, if popups are blocked then the regular link won't work either

Reply:
Odd, Firefox is blocking this one and not the call button... Oh well, good enough

Reply:
try again?